src/store/initialState.js

```diff
@@ -25,5 +25,8 @@ export default {
     singleArticle: {
 
     }
+  },
+  tagsearched_articles: {
```

Comment:
tagsearched_articles does not match any key in the reducers index. Is it being used?

Reply:
Yes it is. Let me look into that.